CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 * don't execute BeforeAll and AfterAll hooks when in dry-run
 * support correct case for `--retry-tag-filter` CLI argument
 
+#### Internals
+
+* Bump Lodash
```

Comment:
Don't think this really belongs in the changelog as not something I expect a user to care about.

Reply:
Done